.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,52 @@
+---
+name: "🐞 Bug report"
+about: Create a report to help us improve
+labels: kind/bug
+---
+
+> Bugs should be filed for issues encountered whilst operating the Tekton Dashboard.
+> You should first attempt to resolve your issues through the community support channels,
+> e.g. `#dashboard` channel in the [Community Slack](https://github.com/tektoncd/community/blob/master/contact.md#slack).
+> Please provide as much detail as possible.
```

Comment:
Can we use a comment here instead of blockquote to keep the resulting issue free of noise?

Reply:
fixed